texascale-org/templates/snippets/core-gl-19-support-align-center.html

```diff
@@ -1,7 +0,0 @@
-<style id="core-gl-19--add">
-    .align-center {
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-    }
-</style>
```

Comment:
(moved to [TACC/Core-CMS:/taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.picture.css](https://github.com/TACC/Core-CMS/blob/task/ECEP-113-support-news/taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.picture.css))